typing_extensions/src_py3/typing_extensions.py

```diff
@@ -1853,31 +1855,18 @@ def __call__(self, *args, **kwargs):
 
         def __getattr__(self, attr):
             # For simplicity we just don't relay all dunder names
-            if self.__origin__ is not None and not (
-                    attr.startswith('__') and attr.endswith('__')
-            ):
+            if self.__origin__ is not None and not _is_dunder(attr):
                 return getattr(self._get_cons(), attr)
             raise AttributeError(attr)
 
         def __setattr__(self, attr, value):
-            if (
-                attr.startswith('__') and attr.endswith('__')
-                or attr.startswith('_abc_')
-            ):
+            if _is_dunder(attr) or attr.startswith('_abc_'):
                 super().__setattr__(attr, value)
             elif self.__origin__ is None:
                 raise AttributeError(attr)
             else:
                 setattr(self._get_cons(), attr, value)
 
-        # We overload this because the super() error message is confusing:
-        # Parametrized generics cannot be used ...
-        def __instancecheck__(self, obj):
-            raise TypeError("Annotated cannot be used with isinstance().")
-
-        def __subclasscheck__(self, cls):
-            raise TypeError("Annotated cannot be used with issubclass().")
```

Comment:
It looks like these are actually needed on _some_ versions, see test failures.

Reply:
OOps, good catch. I really need to setup a better test system on my machine.